test/cets_test_helper.erl

```diff
@@ -0,0 +1,37 @@
+-module(cets_test_helper).
```

Comment:
I must admit I'm not a fan of a general helper, especially if there are specific helpers, but I'm not sure how to fix this.
I've left some comments which may make sense if we reshuffle some testcases, but I also understand if we want to move helper functions out of the `*_SUITE` files altogether - then this module is needed as it is I believe.